src/application_settings/convenience.py

```diff
@@ -0,0 +1,108 @@
+"""Functions that can be called from the application to make life easy."""
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Union
+
+from application_settings.configuring_base import ConfigT
+from application_settings.settings_base import SettingsT
+
+
+def config_filepath_from_cli(
+    config_class: type[ConfigT],
+    parser: ArgumentParser = ArgumentParser(),
+    short_option: str = "-c",
```

Comment:
Why not also have the possibility to set the long option?

Also should the default option have the config/settings class name in the long option.

Related to this, should we support multiple configs and settings? That does not really work with this option

Reply:
Well, for the short option it may occur that you get a conflict with some other option for which you prefer to use that letter; for the long option that is highly unlikely. 
I prefer a fixed default long option, then it is predictable how you can set the path using that without knowing the code, nothing to check / remember.
But I see why one maybe would like to make a different choice. So I will add the long option as a parameter as well.

For now, I don't want to introduce multiple configs / settings. My current idea is that there is one config and/or one settings file. We could introduce an include / import mechanism to support the creation of one parameter file from multiple sources, e.g. as discussed here: https://github.com/toml-lang/toml/issues/397 and as this for json: https://pypi.org/project/json-include/
I will make an issue for this, okay